src/transfers/live.rs

```diff
@@ -107,19 +108,19 @@ impl<T: DuplexTransport + 'static> Future for WatchLiveLogs<T> {
                             handle.spawn(self.process_log(&log));
                         }
                         Ok(Async::Ready(None)) => {
-                            self.tx.close().map_err(move |_| {
-                                web3::Error::from_kind(ErrorKind::Msg("Unable to close sender".to_string()))
-                            })?;
+                            self.tx
+                                .close()
+                                .map_err(move |_| Error::Transport("Unable to close sender".to_string()))?;
```

Comment:
It would be preferable to use our own errors here, rather than just jamming it into Transport